extensions/amp-story/1.0/amp-story-desktop-one-panel.css

```diff
@@ -18,4 +18,22 @@
 
 
 .i-amphtml-story-desktop-one-panel {
+  --large-desktop-ratio: 69 / 116;
+  --small-desktop-ratio: 3 / 4;
+}
+
+.i-amphtml-story-desktop-one-panel amp-story-page {
+  margin: auto auto !important;
```

Comment:
can we do flexbox instead? It has faster layout times usually

Reply:
I think that would require setting `display: flex;` on `amp-story` which could introduce side effects.

It looks like margin is what is currently used. It might be a good option to use for now:
https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/1.0/amp-story-desktop-panels.css#L126